sdk/python/tests/integration/feature_repos/universal/data_sources/bigquery.py

```diff
@@ -42,14 +42,19 @@ def create_offline_store_config(self):
 
     def create_data_sources(
         self,
-        destination: str,
         df: pd.DataFrame,
+        destination: Optional[str] = None,
+        suffix: Optional[str] = None,
         event_timestamp_column="ts",
         created_timestamp_column="created_ts",
         field_mapping: Dict[str, str] = None,
         **kwargs,
     ) -> DataSource:
 
+        assert destination or suffix
```

Comment:
I really think these variables need names so that it's clear to users what a `destination` is and what `suffix` is meant to map to.